### DISCUSSION TITLE
when override method, how do i select these checkboxes.

### DISCUSSION THREAD

**Discussion by PolygonT** [neoclide/coc.nvim#4788] [Q&A]
![image](https://github.com/neoclide/coc.nvim/assets/56947591/1eb84c56-3fc0-4f7f-9c51-01bd70c9f863)

**Comment by fannheyward** [comment]
Use `space` to select.

**Reply by PolygonT** [reply]
i just enter space in the input box.
![image](https://github.com/neoclide/coc.nvim/assets/56947591/c03123b1-a32c-48f9-b4e7-05db641720f7)

**Comment by PolygonT** [comment]
not working with vim，but worked with neovim using `<C-space>`

**Reply by fannheyward** [reply]
https://github.com/neoclide/coc.nvim/issues/4673
